digital_land/phase/convert.py

```diff
@@ -114,16 +114,29 @@ def convert_features_to_csv(input_path):
 
 
 class ConvertPhase(Phase):
-    def __init__(self, path=None, dataset_resource_log=None, custom_temp_dir=None):
+    def __init__(
+        self,
+        path=None,
+        dataset_resource_log=None,
+        custom_temp_dir=None,
+        custom_temp_file=None,
```

Comment:
I would be tempted not to use custom_temp_file for now and just name the file the same as the input file (I know I used custom_temp_file in mine but there should be a way to not use it)

Reply:
Just a warning on the above, the custom_temp_dir is used for something so could be worth double checking what it's used for right now. I haven't spent too long looking at the tempfile package.